src/widget/text-print/text-print.js

```diff
@@ -0,0 +1,66 @@
+import Widget from '../../js/widget';
+
+/**
+ * Clone text input fields value into new print-only element.
+ * This is an unusual way to implement a feature, because it is not an actual widget,
+ * but this is the easiest way to do it.
+ */
+class TextPrintWidget extends Widget {
+    /**
+     * @type string
+     */
+    static get selector() {
+        return '.question > input[type=text]:not(.autocomplete):not(.ignore)';
+    }
+
+    _init() {
+        this.question.classList.add( 'or-text-print-initialized' );
+
+        const printElement = document.createElement( 'div' );
+        printElement.classList.add( this._getClassName() );
+        printElement.classList.add( 'print-only' );
+
+        if ( getComputedStyle( this.element ).order !== '' ) {
+            printElement.style.order = parseInt( getComputedStyle( this.element ).order, 10 ) + 1;
+        }
+        this.element.after( printElement );
+
+        const widget = this.element.parentElement.querySelector( `.${this._getClassName()}` );
+        if ( this.element.value !== '' ) {
+            const widgetText = this.element.value.replace( /\n/g, '<br>' );
+            widget.innerHTML = widgetText;
+        }
+
+        this.element.addEventListener( 'input', event => {
+            const el = event.target;
+            const widgetText = el.value.replace( /\n/g, '<br>' );
+            widget.innerHTML = widgetText;
+        } );
+    }
+
+    _getClassName() {
```

Comment:
Consider whether any of this code is still necessary now that we no longer use this for textareas.... :) Try to simplify as much as you can.